model/application_authentication.go

```diff
@@ -49,8 +49,8 @@ func (aa *ApplicationAuthentication) ToResponse() *ApplicationAuthenticationResp
 	return &ApplicationAuthenticationResponse{
 		ID:                id,
 		AuthenticationUID: aa.AuthenticationUID,
-		CreatedAt:         aa.CreatedAt,
-		UpdatedAt:         aa.UpdatedAt,
+		CreatedAt:         util.DateTimeToRecordFormat(aa.CreatedAt),
+		UpdatedAt:         util.DateTimeToRecordFormat(aa.UpdatedAt),
```

Comment:
So for all of the timestamps that come back in HTTP responses it needs to be `time.RFC1399`, see a v1.0 api response here:
```
$> curl -s -u insights-qa:***** 'https://console.stage.redhat.com/api/sources/v1.0/applications/87919' |jq  
{
  "application_type_id": "3",
  "availability_status": "unavailable",
  "availability_status_error": "Could not enable cloud metering. Error code CG1002. AWS account ID must be unique, but another cloud account with the same AWS account ID already exists.",
  "created_at": "2021-12-08T13:38:39Z",
  "extra": {},
  "id": "87919",
  "source_id": "96814",
  "updated_at": "2021-12-08T13:38:52Z",
  "tenant": "6089719"
}
```

By default we're using `RFC1339Nano` which is fine, but we probably just want to use regular RFC1339.

So I would just make another helper funciton `util.DateTimeToRFC1339(time.Time)`.

List of constants we have here: https://pkg.go.dev/time#pkg-constants

Reply:
function created in util/parser.go